### PR TITLE
Added A simple index

### PR DESCRIPTION
**Pull request by Ank-22**
@gridhead
I realize that it may not be essential, but I believed it was a useful and straightforward addition that could assist new users or visitors to the project.

Please let me know if you would like any changes.

I kept the index simple, but I can create a more detailed one if needed.

I like the idea of using emojis with the section headers; if you'd prefer, I can add them to all sections or remove them from the index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,20 @@
     the development members</b>.
 </p>
 
+
+
+## 📚 Index
```
Comment (gridhead) [inline README.md:71]: The emojis might not be available on platforms like PyPI etc. 

```suggestion
## Index
```

File: README.md [hunk 2]
```diff
@@ -66,6 +66,20 @@
     the development members</b>.
 </p>
 
+
+
```
Comment (gridhead) [inline README.md:70]: Please keep a maximum inter-section space of one newline.

File: README.md [hunk 3]
```diff
@@ -66,6 +66,20 @@
...
+1. **[Screenshots](#screenshots)**
+2. **[Download](#download)**
+3. **[Usage](#usage)**
+4. **[Development](#development)**
+5. **[Contribution](#contribution)**
+6. **[Disclaimer](#disclaimer)**
+
+
+
+
```
Comment (gridhead) [inline README.md:82]: Please keep a maximum inter-section space of one newline.

File: README.md [hunk 4]
```diff
@@ -66,6 +66,20 @@
...
+## 📚 Index
+
+1. **[Screenshots](#screenshots)**
+2. **[Download](#download)**
+3. **[Usage](#usage)**
+4. **[Development](#development)**
+5. **[Contribution](#contribution)**
+6. **[Disclaimer](#disclaimer)**
```
Comment (gridhead) [inline README.md:78]: This does seem to work on GitHub but not on PyPI.

**Review by gridhead — CHANGES_REQUESTED** [review]
Thanks for the impromptu pull request.

Please make the following requested changes.

**Comment by gridhead** [comment]
I do not see much use in this if it does not work on PyPI, as GitHub already seems to have a dedicated semantic outline feature for project documentation. 

{56491442-E368-4A1F-BE19-7419AF948BFA}

**Comment by Ank-22** [comment]
Please continue this discussion on pull request #213, Due to some technical error

**Pull request closed without merging**